src/backend/gl/src/command.rs

```diff
@@ -309,6 +303,34 @@ impl RawCommandBuffer {
             }
         }
     }
+
+    pub(crate) fn bind_attributes(&mut self) {
+        let Cache {
+            ref attributes,
+            ref vertex_buffers,
+            ref vertex_buffer_descs,
+            ..
+        } = self.cache;
+
+        for attribute in attributes {
+            let binding = attribute.binding as usize;
+
+            assert!(vertex_buffer_descs.len() > binding);
+            assert!(vertex_buffers.len() > binding);
+
+            let desc = &vertex_buffer_descs[binding];
```

Comment:
is there a point in `assert!` above if we are just indexing here and thus will get a trap even in release mode?

Reply:
Agreed, it would probably be useful to change these to `error!` instead to let the user know when the vertex buffers or descriptions haven't been bound